pages/src/localdev.ts

```diff
@@ -104,3 +103,6 @@ ESLCarouselPlugins.Dots.register();
 ESLCarouselPlugins.Link.register();
 ESLCarouselPlugins.Touch.register();
 ESLCarouselPlugins.Autoplay.register();
+
+// UI Playground
+import './ui-playground/ui-playground';
```

Comment:
Still a part of the main bundle build
We should make sure it will load separately (async external load or load independently from template itself)